src/main/paradox/contexts/realm.json

```diff
@@ -0,0 +1,12 @@
+{
+  "@context": [
+    "https://bluebrain.github.io/nexus/contexts/resource.json",
+    {
+      "nxv": "https://bluebrain.github.io/nexus/vocabulary/",
```

Comment:
This one is not needed since it is already in https://bluebrain.github.io/nexus/contexts/resource.json